ingresspipe/utils/load_utils.py

```diff
@@ -0,0 +1,42 @@
+import os
```

Comment:
not sure what the best name is for this utils file - it does have some export functionality too... do you think something like io_utils might be ok?

Reply:
Hm..yes, you're right, `io_utils` sounds better. I thought about it for a while, and couldn't come up with a name better than `load_utils` which is why I went with that. But yes, I will change it to `io_utils`.